protocols/yport/yport_net.c

```diff
@@ -85,7 +85,8 @@ yport_net_main(void)
     if (yport_conn == uip_conn)
       yport_conn = NULL;
   }
-  else if (uip_newdata())
+  
+  if (uip_newdata())
```

Comment:
Hi!

>> @@ -85,7 +85,8 @@ yport_net_main(void)
>>      if (yport_conn == uip_conn)
>>        yport_conn = NULL;
>>    }
>> -  else if (uip_newdata())
>> +  
>> +  if (uip_newdata())
> 
> What happens if the statement above has been excecuted and the
> connection has been set to zero?


You are right, maybe a

	&& yport_conn == uip_conn

would be a good addition.

-- 
Michael